browser-test/src/support/admin_programs.ts

```diff
@@ -213,6 +213,20 @@ export class AdminPrograms {
     )
   }
 
+  async expectProgramBlockEditPageWithSuccessToast({
```

Comment:
Personally I would create 2 functions: expectSuccessToast and expectProgramBlockEditPage. They seem to be checking 2 orthogonal things. Also expectSuccessToast can be useful in other places.

Reply:
Done :D